packages/atomic/src/components/search/result-template-components/atomic-quickview-sidebar/atomic-quickview-sidebar.tsx

```diff
@@ -134,28 +143,35 @@ const Keywords: FunctionalComponent<
                   }).format(keyword.occurrences)}
                 </div>
               </div>
-              <div class="flex px-2">
-                <IconButton
-                  partPrefix="sidebar-next"
-                  icon={ArrowDown}
-                  disabled={!wordIsEnabled}
-                  style="text-transparent"
-                  class="border-0"
-                  ariaLabel={i18n.t('next')}
-                  title={i18n.t('next')}
-                  onClick={() => keyword.navigateForward()}
-                />
-                <IconButton
-                  partPrefix="sidebar-previous"
-                  icon={ArrowUp}
-                  disabled={!wordIsEnabled}
-                  style="text-transparent"
-                  class="border-0"
-                  ariaLabel={i18n.t('previous')}
-                  title={i18n.t('previous')}
-                  onClick={() => keyword.navigateBackward()}
-                />
-              </div>
+              <FieldsetGroup
+                label={i18n.t('quickview-navigate-keywords', {
+                  occurrences: keyword.occurrences,
+                  keyword: sanitize(keyword.text),
+                })}
+              >
+                <div class="flex px-2">
+                  <IconButton
+                    partPrefix="sidebar-next"
+                    icon={ArrowDown}
+                    disabled={!wordIsEnabled}
+                    style="text-transparent"
+                    class="border-0"
+                    ariaLabel={i18n.t('next')}
+                    title={i18n.t('next')}
+                    onClick={() => keyword.navigateForward()}
```

Comment:
It seems overkill, since the "selection" is not even something you can act on ? 
ie: You can do nothing with the selection, it's purely a visual trick that changes the color of the background of individual keyword.

Tempted to just leave it as is for now ?